zerver/openapi/zulip.yaml

```diff
@@ -17103,14 +17100,21 @@ components:
           type: string
     SuccessDescription:
       description: |
+        **Note**: If any parameters sent in the request are not supported
+        by this endpoint, a successful JSON response will include
+        an [ignored_parameters_unsupported][ignored_params] array.
+
         A typical successful JSON response may look like:
+
+        [ignored_params]: /api/rest-error-handling#ignored-parameters
```

Comment:
Should these entries have a `**Changes**` entry?

Reply:
I guess probably anyone who needs that information will probably click the link, so maybe it's not necessary?